syncer/ddl_test.go

```diff
@@ -415,14 +431,31 @@ func (s *testSyncerSuite) TestResolveOnlineDDL(c *C) {
 	ec := eventContext{
 		tctx: tctx,
 	}
-
+	cluster, err := mock.NewCluster()
+	c.Assert(err, IsNil)
+	c.Assert(cluster.Start(), IsNil)
+	mysqlConfig, err := mysql.ParseDSN(cluster.DSN)
+	c.Assert(err, IsNil)
+	mockClusterPort, err := strconv.Atoi(strings.Split(mysqlConfig.Addr, ":")[1])
+	c.Assert(err, IsNil)
+	dbCfg := config.GetDBConfigFromEnv()
+	dbCfg.Port = mockClusterPort
+	dbCfg.Password = ""
+	cfg := &config.SubTaskConfig{
+		From:       dbCfg,
+		To:         dbCfg,
+		ServerID:   101,
+		MetaSchema: "test",
+		Name:       "syncer_ut",
+		Mode:       config.ModeIncrement,
+		Flavor:     "mysql",
+	}
```

Comment:
Does it work same as L511-520? Do you consider merging into one function?

Reply:
added in https://github.com/pingcap/dm/pull/2120/commits/3d6828722476ab5a83f9478b03cb54b86c8bfdd3